module-2/backend/week_two.md

```diff
@@ -25,21 +33,68 @@ end
 ```
 
 Now how would you find the owner of the team with an id of 4?
+Assuming a foreign key had been added to the teams table:
+Team.find(4).owner
 
 5. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
+
+Teachers have many students
+
+Teacher                  Student
+id -----------           id
+name          |          name
+class_name    |_________ teacher_id
+
+
 6. Define foreign key, primary key, and schema.
+A primary key is usually the id and Active record assigns it for you, although you can change it. A foreign key is the id for another table that you need in order to make relationships. A schema is a picture that shows all of the relationships between your tables.
```

Comment:
I like the nuance in this answer re: changing the primary key. Shows a good understanding of this concept.